github/github_test.go

```diff
@@ -1136,6 +1141,67 @@ func TestDo_rateLimit(t *testing.T) {
 	}
 }
 
+func TestDo_rateLimitCategory(t *testing.T) {
+	tests := []struct {
+		method   string
+		url      string
+		category rateLimitCategory
+	}{
+		{
+			http.MethodGet,
```

Comment:
This is a tiny nit, and if you don't want to do this, I'm fine...
but in this repo we prefer to label the fields for clarity-at-a-quick-glance (e.g. `method: http.MethodGet,`, etc.).
I'm going to go ahead and approve, but if you want to update, great.

Reply:
sure, np, I'll do that. 
p.s. Ubuntu for desktop, RHEL/fedora-based for server 🙃